content/docs/started/On-Prem/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "On-Prem Installation"
+description = "Get started with Kuberflow On-Prem"
```

Comment:
s/Kuberflow/Kubeflow

Reply:
Good catch! Must have copy/pasted this everywhere